learn/index.html

```diff
@@ -186,6 +188,29 @@ <h3>Differences</h3>
                     <dd>APRS is quite a bit more difficult to use than Meshtastic. APRS does not have any standardized user interface – there are dozens of software packages available. Meshtastic has a standardized iPhone and Android app which is easy to understand for new users.</dd>
                 </dl>
             </section>
+            <section id="bridger">
+                <h2>Network Observability</h2>
+                <p><a href="https://github.com/austinmesh/bridger">Bridger</a> powers the backend of our network statistics dashboard. The project is maintained by the Austin Mesh community and written in Python. With Bridger, we can ingest MQTT packets from contributing gateways, parse the packets into <a href="https://github.com/influxdata/influxdb" >InfluxDB</a>, and represent the network statistics with <a href="https://grafana.com/">Grafana</a>.</p>
+                <dl>
+                    <dt>How to view the Austin Mesh network statistics dashboard?</dt>
+                    <dd>Our network statistics dashboard can be viewed by visiting <a href="https://graphs.austinmesh.org">graphs.austinmesh.org</a>. Feel free to explore the data and see the coverage, device metrics, node information, and more.</dd>
+                    <dt>Why is my node not listed?</dt>
+                    <dd>Newer versions of the Meshtastic firmware have a setting called "Okay to MQTT" under the LoRa settings. This is a polite request to a device running the official firmware on whether or not packets from your node can be forwarded to MQTT brokers by other nodes. Please ensure that this is enabled to have your node reported on the network statistics dashboard. Please see the <a href="/join/#best-practices">best practices</a> page for more information on our recommended configuration for nodes.</dd>
+                    <dt>How can I contribute data to Bridger?</dt>
+                    <dd>To contribute data to Bridger, you will need a node that can connect to the MQTT server through the internet, like a Heltec V3. Ideally, contributing nodes should be connected 24/7. Your node will need credentials, which can be requested on the Austin Mesh discord by running the <b>/bridger-mqtt-request-account</b> command and supplying your 8-digit node ID. Be sure to store your username and password in a safe place. Once you have your credentials, open the MQTT settings and configure it as follows:<br><br>
```

Comment:
```suggestion
                    <dd>To contribute data to Bridger, you will need a node that can connect to the MQTT server through the internet either directly(ESP32) or through a connected client with MQTT>Proxy to Client enabled. Ideally, contributing nodes should be connected 24/7. Your node will need credentials, which can be requested on the Austin Mesh discord by running the <b>/bridger-mqtt-request-account</b> command and supplying your 8-digit node ID. Be sure to store your username and password in a safe place. Once you have your credentials, open the MQTT settings and configure it according to the table below.
```

Reply:
The following table is going to be another reason to not use the `dl` element and instead use headings. Tables can't go inside a `dd` element, actually no other html elements can, except for `div`.